packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+
+/**
+ * An implementation of ITelemetryBaseLogger that routes Fluid telemetry
+ * events to Azure App Insights using the App Insights trackEvent API
+ * @public
+ */
+export class FluidAppInsightsLogger implements ITelemetryBaseLogger {
+	/**
+	 * The Azure ApplicationInsights client utilized by this logger
+	 */
+	protected readonly baseLoggingClient: ApplicationInsights;
+
+	public constructor(client: ApplicationInsights) {
+		this.baseLoggingClient = client;
+		this.baseLoggingClient.loadAppInsights();
```

Comment:
[Still think](https://github.com/microsoft/FluidFramework/pull/14848#discussion_r1153506090) that we should tell consumers that they need to give us a fully initialized app insights client. If there's a way to check if `loadAppInsights()` was already called on it, I'd even suggest we check and throw an error if it hasn't been (i.e. the client is not in the state we expect it).

I tried to look at the code; it would seem calling `loadAppInsights()` twice won't throw or anything, but I'd still prefer that we completely get out of the business of setting up the client, since there's nothing we need configured in a particular way.

Reply:
I tested this and calling it twice seemed to be fine but there could perhaps be some kind of a side effect so I can remove this and update the documentation accordingly.